{
  "app": {
    "error": {
      "title": "Máy chủ đang gặp sự cố",
      "description": "Chúng tôi đã gặp phải một lỗi không mong đợi. Đội ngũ của chúng tôi đã được thông báo và chúng tôi đang làm việc để khắc phục. Vui lòng thử lại sau.",
      "retry": "Thử lại"
    },
    "not-found": {
      "title": "Không tìm thấy trang",
      "description": "Trang này không tồn tại trên trang web của chúng tôi.",
      "backToHome": "Quay về trang chủ"
    }
  },
  "features": {
    "globals": {
      "locale-switcher": {
        "en": "Tiếng Anh",
        "vi": "Tiếng Việt"
      },
      "theme-switcher": {
        "light": "Sáng",
        "dark": "Tối",
        "auto": "Tự động",
        "system": "Hệ thống"
      }
    }
  },
  "lib": {
    "axios": "Lỗi gọi API: "
  }
}
